### DISCUSSION TITLE
Add support for the new formats

### DISCUSSION THREAD

**Discussion by edscodeblenders** [firebase/flutterfire#4783] [Feature request]
Admob has recently released 2 new formats in beta:

1. App Open: https://developers.google.com/admob/android/app-open-ads
2. Rewarded Interstitials: https://developers.google.com/admob/android/rewarded-interstitial

Google is recommending users to develop apps with Flutter, but then we don't have all the monetization tools available for native development. It would be great if you can support these 2 new formats.

**Comment by oliverbytes** [comment]
Patiently waiting for these new and awesome ad formats!
